apps/web/core/query/keys/index.ts

```
@@ -360,7 +360,29 @@ export const queryKeys = {
+			projectIds?: string[] | null | undefined,
+			taskIds?: string[] | null | undefined,
+			status?: string[] | null | undefined
+		) =>
+			[
+				'timesheet',
+				'logs',
+				...(tenantId ? [tenantId] : []),
+				...(organizationId ? [organizationId] : []),
+				...(startDate ? [startDate] : []),
+				...(endDate ? [endDate] : []),
+				...(employeeIds && employeeIds.length ? [employeeIds.join(',')] : []),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)